integration-tests/integration/reviewer/view-not-completed-incidents.spec.js

```diff
@@ -162,45 +143,17 @@ context('A use of force reviewer can view all incidents at the current agency',
       ],
     })
```

Comment:
can get rid of some of these report seeding - we only had 3 so we could do a bit of testing on search. As there isn't search any more its safe to get rid of them.

Reply:
Yes. There isn't a need for the 'filter' test because there is no search anymore. Test and seeding removed.